package.json

```diff
@@ -11,10 +11,13 @@
 	],
 	"dependencies": {
 		"@directus/sdk": "10.2.0",
-		"gatsby-source-filesystem": "4.13.0",
-		"gatsby-source-graphql": "4.13.0",
 		"ms": "2.1.3"
 	},
+	"peerDependencies": {
+		"gatsby-source-filesystem": "4||5",
+		"gatsby-source-graphql": "4||5",
+		"eslint": "7||8"
```

Comment:
Sorry for this post review, but why `eslint` as a peer dependency?

Reply:
I will check it out during the weekend and then will add another comment in here.